apps/web/src/features/positions/components/PositionsUnavailable/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image'
+import { Paper, Typography } from '@mui/material'
+import DefiImage from '@/public/images/balances/defi.png'
+
+// This component is displayed when the positions feature flag IS enabled,
+// but the API does not return data from CGW (Client Gateway).
+const PositionsUnavailable = () => {
+  return (
+    <Paper elevation={0} sx={{ p: 3, textAlign: 'center' }}>
+      <Image src={DefiImage} alt="Defi illustration" width={100} height={100} />
```

Comment:
AFAIR we never used the Image component in this codebase because we're not using server-side rendering. Please check if I'm correct, replace with a regular `<img>`.